docs/lib/project-setup/fragments/flutter/null-safety/null-safety.md

```diff
@@ -0,0 +1,19 @@
+**Amplify Flutter and Null Safety**
+
+
+The Amplify Flutter library supports [Dart null safety](https://dart.dev/null-safety) beginning with version 0.2.0.  
+
+|                                	                     | amplify-flutter 0.1.6 and below   	| amplify-flutter 0.2.0 and above  	|
+|-------------------------------	|---------------------------------	|---------------------------------	|
+| Null Safe App     	                         | Not Supported                             	| Supported                                    	|
+| Non Null Safe App w/ flutter v2  | Supported                                    	| Supported                                    	|
+| Non Null Safe App w/ flutter v1 	| Supported                                    	| Not Supported                                    	|
+
+
+
+**Using Generated Codegen Models with Null Safety**
+
+
+The latest version of the Amplify CLI can generate Dart models with or without null safety using the `codegen models` command. To generate null safe models, make sure that the `pubspec.yaml` file at the root of your projects defines a Dart SDK version of 2.12.0 or greater, and make sure that the `enableDartNullSafety` [feature flag](https://docs.amplify.aws/cli/reference/feature-flags) is set to true in your `amplify/cli.json` file. You may need to update your Amplify CLI to version 5.1.0 or higher.
```

Comment:
You may need to -> You will need to update your Amplify CLI to version 5.1.0 or higher

Reply:
Consider having these instructions as separate bullet points or a numbered list. 
Also we need to have the instructions.